index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ToDo List</title>

```

Comment:
나중에는 스타일도 적용해보는 것을 추천드립니다!
```suggestion
    <title>ToDo List</title>
    <link rel="stylesheet" href="style.css" />
```
```suggestion
    <title>ToDo List</title>

```